tests/sls/homu/valid_builders.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os.path
+import subprocess
+import toml
+
+from tests.util import Failure, Success
+
+
+def are_builders_valid(buildbot, homu_cfg, buildertype):
+    for builder in homu_cfg[buildertype]:
+        if builder not in buildbot:
+            diff = list(set(homu_cfg[buildertype]) - set(buildbot))
+            diff_print = ''
+            if len(diff) != 0:
+                diff_print = 'Difference: {}'.format(diff)
+
+            fail = Failure(
+                'Homu "{}" config isn\'t sync with buildbot config'
```

Comment:
nit: sync -> synced
Also, using double quotes for this string will make it easier to read (no need to escape the apostrophe).